pkg/configmap/configmap.go

```diff
@@ -128,27 +128,30 @@ func (s *Sync) Run() (chan struct{}, error) {
 		return nil, err
 	}
 	quit := make(chan struct{})
-	source := cache.NewListWatchFromClient(
-		client,
-		"configmaps",
-		v1.NamespaceAll,
-		fields.Everything())
-	store, controller := cache.NewInformer(
-		source,
-		&v1.ConfigMap{},
-		time.Second*60,
-		cache.ResourceEventHandlerFuncs{
-			AddFunc:    s.add,
-			UpdateFunc: s.update,
-			DeleteFunc: s.delete,
-		})
-	for _, obj := range store.List() {
-		cm := obj.(*v1.ConfigMap)
-		if match, isPolicy := s.matcher(cm); match {
-			s.syncAdd(cm, isPolicy)
+
+	for _, namespace := range namespaces {
+		source := cache.NewListWatchFromClient(
+			client,
+			"configmaps",
+			namespace,
```

Comment:
Users can deploy kube-mgmt with `--policies=*` in which case configmaps in all namespaces will be replicated into OPA (this is used in conjunction with --require-policy-label` to filter configmaps.)

Could you add a flag to kube-mgmt so that users can tell it to watch specific namespaces? Otherwise this change will not be backwards compatible.

Reply:
I'm a bit afraid that adding another parameter would maybe clutter the UI too much? Note that the help for `--policies` currently says:
```
 --policies stringSlice                automatically load policies from these namespaces (default [opa,kube-federation-scheduling-policy]) 
```
Can we not simply parse the `--policies=` parameter and in case of `*` use `v1.NamespaceAll`? Something like
```golang
	if namespaces[0] == "*" {
		namespaces[0] = v1.NamespaceAll
		namespaces = namespaces[0:1]
	}
```

Reply:
This looks fine. I would probably check whether the slice contained `*` at all but this will work.